react/features/base/connection/actions.native.ts

```diff
@@ -141,12 +141,16 @@ function _connectionWillConnect(connection: Object) {
     };
 }
 
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Note to self: add this ignore to our eslint config to avoid this problem: `'no-unused-vars': [ 'error', { 'argsIgnorePattern': '^_' } ],`